RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64Bitmap.java

```diff
@@ -596,13 +596,35 @@ public void add(long... dat) {
     }
   }
 
+  /**
+   * Verifies that a range is correct: being should strictly be less than end and
+   * end cannot be zero.
+   * @param begin inclusive beginning of range
+   * @param end exclusive ending of range
+   */
+  void rangeSanityCheck(final long begin, final long end) {
+    if (end == 0 || begin == end) {
```

Comment:
Can you explain why this cannot be done with `Long.compareUnsigned(long x, long y).` Or, at least, be integrated in the `add` routine?

Reply:
Didn't know `Long` had such a helper method. Adopting your suggestion.